src/github.com/getlantern/launcher/launcher_windows.go

```diff
@@ -8,13 +8,16 @@ import (
 	"github.com/getlantern/golog"
 )
 
+const (
+	runDir = `Software\Microsoft\Windows\CurrentVersion\Run`
```

Comment:
Instead of converting this constant into a variable scoped to a single function I made it an unexported constant by making the first char lowercase. It would be easier to become aware of its *constantness* this way.